std/range/package.d

```diff
@@ -9213,3 +9221,292 @@ if (is(typeof(fun) == void) || isSomeFunction!fun)
 
     auto r = [1, 2, 3, 4].tee!func1.tee!func2;
 }
+
+/**
+Extend the length of the input range $(D r) by padding out the start of the
```

Comment:
I think new additions to the doc should at best use backquotes instead of `$(D )`.